src/current/molt/migration-overview.md

```diff
@@ -0,0 +1,136 @@
+- [Failback]({% link molt/molt-fetch.md %}#fail-back-to-source-database) to replicate changes from CockroachDB back to the original source via a secure changefeed
+
+### Verify
+
+[MOLT Verify]({% link molt/molt-verify.md %}) checks for data and schema discrepancies between the source database and CockroachDB. It performs:
+
+- Table structure verification
+- Column definition verification
+- Row-level data verification
+{% comment %}- Continuous, live, or one-time verification{% endcomment %}
+
+## Migration modes
+
+MOLT Fetch supports [multiple data migration modes]({% link molt/molt-fetch.md %}#fetch-mode). These can be combined based on your testing and cutover strategy.
+
+|                     Mode                    |                                 Description                                  |                                               Best For                                               |
+|---------------------------------------------|------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| `--mode data-load`                          | Performs one-time load of source data into CockroachDB                       | Testing, migrations with planned downtime, [phased migrations]({% link molt/migrate-in-phases.md %}) |
+| `--mode data-load-and-replication`          | Loads source data and starts continuous replication from the source database | [Migrations with minimal downtime]({% link molt/migrate-to-cockroachdb.md %})                        |
+| `--mode replication-only`                   | Starts replication from a previously loaded source                           | [Phased migrations]({% link molt/migrate-in-phases.md %}), post-load sync                            |
+| `--mode failback`                           | Replicates changes on CockroachDB back to the original source                | [Rollback scenarios]({% link molt/migrate-failback.md %})                                            |
+| `--mode export-only` / `--mode import-only` | Separates data export and import phases                                      | Large-scale migrations, custom storage pipelines                                                     |
+| `--direct-copy`                             | Loads data without intermediate storage using `COPY FROM`                    | Local testing, limited infra environments                                                            |
+
+## Migrations with minimal downtime
+
+MOLT simplifies and streamlines the following migration patterns that use a replication stream to minimize downtime. Rather than load all data into CockroachDB during a planned downtime window, you perform an initial data load and continuously replicate any subsequent changes to CockroachDB. Writes are only briefly paused to allow replication to drain before final cutover.
```

Comment:
should we talk about this pause and how it pretains to how long your downtime will be? Since it depends on your write traffic and replication lag